### PR TITLE
Initial Code

### PR DESCRIPTION
**Pull request by ddriddle**

### CODE REVIEW

**Review by mpitcel — APPROVED** [review]

**Comment by edthedev** [comment]
First test is passing!

Should probably add issues to split functionality and tests along VCRpy and PyTest lines.

File: tests/test_vcr_cleaner.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+# To record, `export VCR_RECORD=True`
+
+CASSETTE_ENDPOINT = 'https://cybersecurity.illinois.edu'
+
+# TODO: Add unit tests. 
+
+# TODO: Think about how to get rid of need 
+# to export VCR_RECORD and deleting the YAML
+def test_with_vcr(cassette):
```
Comment (edthedev) [inline tests/test_vcr_cleaner.py:25]: This whole test is an integration test at the moment, as is more important as documentation of how to use this library than as a test.

Should we move this into a 'integration' tests folder, or try harder to make it not read the file on the drive?

**Comment by edthedev** [comment]
My other question for the team, this morning, is if we're at the point we can start refactoring our soar app template repo to use this code base. I think we're close, and we might let that goal guide our next changes.

File: tests/test_vcr_cleaner.py (hunk 1 above)
Comment (edthedev) [reply tests/test_vcr_cleaner.py:25]: Update per discussion - let's put a # TODO: test_with_vcr should become a DocTest(s) on this for later when we're building the documentation.

**Review by edthedev — APPROVED** [review]

**Comment by edthedev** [comment]
Let's add a workflow file so the unit tests will run.

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# This workflow tests the code in a production-like environment on our internal dev machine.
```
Comment (ddriddle) [inline .github/workflows/deploy.yml:1]: Remove comment

```suggestion
```

**Review by tzturner — APPROVED** [review]

File: src/vcr_cleaner.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+from vcr.serializers import yamlserializer
+
+# CASSETTE_USERNAME = "FAKE_USERNAME"
+# CASSETTE_PASSWORD = "FAKE_PASSWORD"
+# CASSETTE_ENDPOINT = "cybersecurity.illinois.edu/robots.txt"
+
+# To record, `export VCR_RECORD=True`
+VCR_RECORD = "VCR_RECORD" in os.environ
+
+# TODO: Split this file, and tests between functionality that is VCR only
```
[the remark this answers is not in the document]
Comment (edthedev) [reply src/vcr_cleaner.py:17]: I made #2 to track this.

**Review by edthedev — APPROVED** [review]

**Pull request merged**